src/main/g8/src/main/scala/com/example/WebServerHttpApp.scala

```diff
@@ -0,0 +1,23 @@
+package com.example
+
+import akka.http.scaladsl.marshallers.xml.ScalaXmlSupport
+import akka.http.scaladsl.server.{ HttpApp, Route }
+
+/**
+  * Server will be started calling `WebServerHttpApp.startServer("localhost", 8080)`
+  * and it will be shutdown after pressing return.
+  */
+object WebServerHttpApp extends HttpApp with App with ScalaXmlSupport {
+  // Routes that this WebServer must handle are defined here
+  def route: Route = pathEndOrSingleSlash { // Listens to the top `/`
```

Comment:
I have no strong preference, usually if it fits on the line, I put it there. However, I see your point, putting it next line might be easier to see that it's a route directive.